notebooks/curation_api/README.md

```diff
@@ -0,0 +1,24 @@
+# Discover API notebooks
+
+The Discover API notebooks are pre-baked scripts to facilitate use of the 
+[Discover API](https://api.cellxgene.cziscience.com/curation/ui/#/), which itself supports 
+curation flow interactions with the [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) tool.
+
```

Comment:
`These notebooks facilitate use of the [Discover API](https://api.cellxgene.cziscience.com/curation/ui/#/) which enables data consumers to query for public collections and datasets available in [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) and to download individual datasets. `